README.md

```diff
@@ -23,6 +23,14 @@ Add a marker to the tests that will be picked up to be added to the run.
 	@testrail('C1234', 'C5678')
 	def test_foo():
 		# test code goes here
+	
+    # OR	
+	
+	from pytest_testrail.plugin import pytestrail
+	
+	@pytestrail.case('C1234', 'C5678')
```

Comment:
Would be happy just to move to one decorator, for anyone updating it would be rather simple find and replace.
What do you think ?

Reply:
Yea having just one would be better for sure. For now, I changed the `pytest_testrail.testrail` decorator to just proxy to the `pytest_testrail.pytestrail.case` decorator and added a deprecation warning to give people a heads up.

```
========================================================================== warnings summary ===========================================================================
tests/test_plugin.py::test_get_testrail_keys
  /Users/samfonseca/dev/git/pytest-testrail/.tox/py3-test/lib/python3.6/site-packages/pytest_testrail/plugin.py:55: DeprecatedTestDecorator: pytest_testrail: the @testrail decorator is deprecated and will be removed. Please use the @pytestrail.case decorator instead.
    warnings.warn(deprecation_msg, DeprecatedTestDecorator)

-- Docs: http://doc.pytest.org/en/latest/warnings.html
================================================================ 8 passed, 1 warnings in 0.20 seconds =================================================================
```